packages/utils/src/loader/svg-loader.ts

```diff
@@ -1,5 +1,12 @@
+import { CacheIndexedDBStorage, LocalCache } from '../cache.js';
 import { CDNLoader } from './cdn-loader.js';
 
+// Storage
+const storage = new CacheIndexedDBStorage({ dbName: 'ef', version: 1, storeName: 'svg-loader' });
```

Comment:
I may suggest to use more unique dbName to avoid any potential clash

Reply:
Due to [Trem feedback](https://github.com/Refinitiv/refinitiv-ui/pull/412#pullrequestreview-1066797912), the storage will receive only name. That name will be combine with shared static prefix for both database name and store name. Version will be fit to 1.  So those concept will avoid redundant database and avoid indexeddb clash.